include/utils.h

```diff
@@ -152,27 +151,49 @@ class AlignedFileReader;
 namespace diskann {
   static const size_t MAX_SIZE_OF_STREAMBUF = 2LL * 1024 * 1024 * 1024;
 
+  inline void print_error_and_terminate(std::stringstream& error_stream) {
+    diskann::cerr << error_stream.str() << std::endl;
+    throw diskann::ANNException(error_stream.str(), -1, __FUNCSIG__, __FILE__,
+                                __LINE__);
+  }
+
+  inline void report_memory_allocation_failure() {
+    std::stringstream stream;
+    stream << "Memory Allocation Failed.";
+    print_error_and_terminate(stream);
+  }
+
+  inline void report_misallignment_of_requested_size() {
```

Comment:
typo.. misalignment instead of misallignment (single l)

Reply:
Corrected.